test/39-compose-images.spec.ts

```diff
@@ -0,0 +1,530 @@
+				Labels: {
+					'io.balena.architecture': 'aarch64',
+					'io.balena.livepush-image': '1',
+					'io.balena.qemu.version': '4.0.0+balena-aarch64',
+				},
+				ParentId:
+					'sha256:4acb1652178e72c0b6143a08225e0df5ef74b338a0c9e2ca4cd261339f4f0431',
+				RepoDigests: null,
+				RepoTags: ['livepush-supervisor:11.12.11'],
+				SharedSize: -1,
+				Size: 501352885,
+				VirtualSize: 501352885,
+			},
+			'sha256:272c54e27104cf6c2153538165dba3c29b58ae35837b47134fa55b53ddb61154': {
+				Containers: -1,
+				Created: 1599181227,
+				Id:
+					'sha256:272c54e27104cf6c2153538165dba3c29b58ae35837b47134fa55b53ddb61154',
+				Labels: {
+					'io.balena.architecture': 'aarch64',
+					'io.balena.device-type': 'jetson-tx2',
+					'io.balena.qemu.version': '4.0.0+balena2-aarch64',
+				},
+				ParentId: '',
+				RepoDigests: [
+					'registry2.balena-cloud.com/v2/459bffd69101d788cc3e0722a012878a@sha256:3cc9231cf0e117585e53ebfa6bf9f75a9a4eaa371fb82c21ab9bca8fe0d5c3e3',
+					'registry2.balena-cloud.com/v2/b0cfe2b1e8c5ab3b6da23f0bd92045b4@sha256:ba5f6d1849c63c8d0f11f35fce694464240002d2c3732898935bf0fedf451063',
+				],
+				RepoTags: [
+					'registry2.balena-cloud.com/v2/459bffd69101d788cc3e0722a012878a:delta-80ed841a1d3fefa9',
+					'registry2.balena-cloud.com/v2/b0cfe2b1e8c5ab3b6da23f0bd92045b4:delta-532f970c60decb81',
+				],
+				SharedSize: -1,
+				Size: 217024558,
+				VirtualSize: 217024558,
+			},
+			'sha256:25d6abae14f08de6b80f9d95003e674598738959a535a2f21be34c03675ebd02': {
+				Containers: -1,
+				Created: 1599146808,
+				Id:
+					'sha256:25d6abae14f08de6b80f9d95003e674598738959a535a2f21be34c03675ebd02',
+				Labels: { 'io.balena.qemu.version': '4.0.0+balena-aarch64' },
+				ParentId: '',
+				RepoDigests: [
+					'balena/aarch64-supervisor@sha256:e7e3b166e855f4c113a67bc528b4ef77408d05e280052f35452f3e2cd7b5322b',
+				],
+				RepoTags: ['balena/aarch64-supervisor:v11.14.0'],
+				SharedSize: -1,
+				Size: 72305850,
+				VirtualSize: 72305850,
+			},
+			'sha256:60783b8688d395f9b4ce4b288d941244d1b0a4c43114ba980acd012ccffc6b53': {
+				Containers: -1,
+				Created: 1585746557,
+				Id:
+					'sha256:60783b8688d395f9b4ce4b288d941244d1b0a4c43114ba980acd012ccffc6b53',
+				Labels: { 'io.balena.qemu.version': '4.0.0+balena-aarch64' },
+				ParentId: '',
+				RepoDigests: [
+					'balenalib/aarch64-alpine-supervisor-base@sha256:6eb712fc797ff68f258d9032cf292c266cb9bd8be4cbdaaafeb5a8824bb104fd',
+				],
+				RepoTags: ['balenalib/aarch64-alpine-supervisor-base:3.11'],
+				SharedSize: -1,
+				Size: 14503741,
+				VirtualSize: 14503741,
+			},
+			'sha256:a29f45ccde2ac0bde957b1277b1501f471960c8ca49f1588c6c885941640ae60': {
+				Containers: -1,
+				Created: 1578015959,
+				Id:
+					'sha256:a29f45ccde2ac0bde957b1277b1501f471960c8ca49f1588c6c885941640ae60',
+				Labels: null,
+				ParentId: '',
+				RepoDigests: null,
+				RepoTags: ['balena-healthcheck-image:latest'],
+				SharedSize: -1,
+				Size: 9136,
+				VirtualSize: 9136,
+			},
+		};
```

Comment:
This is great, the only thing I'd request is moving these huge data payloads out to separate JSON files, to keep the file a little cleaner.